src/DssDeploy.t.sol

```diff
@@ -205,7 +205,8 @@ contract DssDeployTest is DssDeployTestBase {
 
     function testBite() public {
         deploy();
-        this.file(address(cat), "ETH", "lump", 1 ether); // 1 unit of collateral per batch
+        this.file(address(cat), "ETH", "lump", rad(200 ether)); // 200 DAI max per batch
```

Comment:
Doesn't this need to become `dunk`? Ditto below.

Reply:
Or are we waiting on the later changes being merged to `dss` master before we do that?

Reply:
Yeah, since we changed the meme, this PR will have to be updated.